tests/base.py

```diff
@@ -36,10 +46,33 @@
 cyclonedx_lib_name: str = 'cyclonedx-python-lib'
 cyclonedx_lib_version: str = version(cyclonedx_lib_name)
 single_uuid: str = 'urn:uuid:{}'.format(uuid4())
+schema_directory = os.path.join(os.path.dirname(__file__), '../cyclonedx_py/schema')
 
 
 class BaseJsonTestCase(TestCase):
 
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.FIXTURES_DIRECTORY = path.join(path.dirname(__file__), 'fixtures')
+
+    def assertValidAgainstSchema(self, bom_json: str, schema_version: SchemaVersion) -> None:
+        if sys.version_info >= (3, 7):
+            schema_fn = os.path.join(
+                schema_directory,
+                f'bom-{schema_version.name.replace("_", ".").replace("V", "")}.schema.json'
+            )
+            with open(schema_fn) as schema_fd:
+                schema_doc = json.load(schema_fd)
+
+            try:
+                json_validate(instance=json.loads(bom_json), schema=schema_doc)
+            except ValidationError as e:
+                self.assertTrue(False, f'Failed to validate SBOM against JSON schema: {str(e)}')
+
+            self.assertTrue(True)
+        else:
+            self.assertTrue(True, 'JSON Schema Validation is not possible in Python < 3.7')
```

Comment:
use unittest's `skipif` https://docs.python.org/3/library/unittest.html?highlight=skipif#unittest.skipIf
to skip tests. sont write conditional foo